lib/test-requirements.txt

```diff
@@ -26,7 +26,7 @@ testfixtures
 # We pin playwright to a minor version to avoid new browser versions
 # breaking our CI. But this version should be updated regularly as soon as
 # new playwright versions are released.
-playwright==1.42.*
+playwright==1.44.*
```

Comment:
Update playwright version, since we anyways have to update and check lots of screenshots.